lib/Progress/Progress.test.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import {
+  screen, render,
+} from '@testing-library/react';
+
+import '../../test/jest/__mock__';
+import '../../test/jest/__new_mock__';
+
+import { Progress } from './Progress';
+
+describe('Progress', () => {
+  const currentValue = 10;
+  const totalValue = 100;
+  const renderDefaultProgress = () => render(
+    <Progress
+      current={currentValue}
+      total={totalValue}
+    />
+  );
+
+  it('should display progress component', () => {
+    renderDefaultProgress();
+
+    expect(screen.getByTestId('progress-bar')).toBeVisible();
+  });
+
+  it('should display correct current progress position', () => {
+    renderDefaultProgress();
+
+    expect(screen.getByTestId('progress-current')).toHaveAttribute('style', `width: ${totalValue / currentValue}%;`);
+  });
+
+  it('should display correct progress info', () => {
+    renderDefaultProgress();
+
+    expect(screen.getByText(`${totalValue / currentValue}%`)).toBeVisible();
+  });
+
+  describe('rendering Progress with "messagedPercentage" progress info type', () => {
+    it('should display correct progress info', () => {
+      const testMessage = 'TestMessage';
+
+      render(<Progress
```

Comment:
looks like a copypaste of renderDefaultProgress, change it to renderProgress and support props with default values

Reply:
done.

> looks like a copypaste of renderDefaultProgress, change it to renderProgress and support props with default values

done.